index.html

```diff
@@ -24,11 +24,255 @@
+                    alt="secure icon">
+                    <div class="flex-column">
+                        <h5>Secure Payment</h5>
+                        <span class="p-xs">Safe and secured payments</span>
+                    </div>
+                </div>
+            </div>
+        </div>
+
+        <div class="flex-column flex-center mg-2">
+            <div class="flex-layout">
+                <div class="category-container">
+                    <a href="productListing.html" class="flex-column flex-center">
+                        <img class="category-img" src="assets/images/dog.png"
+                        alt="dog icon">
+                        <span class="txt-bold">Dogs</span>
+                    </a>
+                </div>
+                <div class="category-container">
+                    <a href="productListing.html" class="flex-column flex-center">
+                        <img class="category-img" src="assets/images/cat.png"
+                        alt="cat icon">
+                        <span class="txt-bold">Cats</span>
+                    </a>
+                </div>
+                <div class="category-container">
+                    <a href="productListing.html" class="flex-column flex-center">
+                        <img class="category-img" src="assets/images/bird.png"
+                        alt="bird icon">
+                        <span class="txt-bold">Birds</span>
+                    </a>
+                </div>
+                <div class="category-container">
+                    <a href="productListing.html" class="flex-column flex-center">
+                        <img class="category-img" src="assets/images/fish-tank.png"
+                        alt="fish icon">
+                        <span class="txt-bold">Fish</span>
+                    </a>
+                </div>
+                <div class="category-container">
+                    <a href="productListing.html" class="flex-column flex-center">
+                        <img class="category-img" src="assets/images/bunny.png"
+                        alt="bunny icon">
+                        <span class="txt-bold">Bunny</span>
+                    </a>
+                </div>
+            </div>
+            
+        </div>
+
+        <div class="flex-column glass-container product-showcase">
+            <div class="flex-column flex-center">
+                <h3>Trending Products</h3>
+                <div class="flex-layout flex-wrap flex-center">
+                    <div class="flex-col-space-btwn mg-1 product-card">
+                        <img class="product-image-1" src="assets/images/treat-dispenser.png"
+                        alt="product image">
+                        <span class="card-badge">New</span>
+                        <button class="card-like"><i class="fa fa-heart-o"></i></button>
+                        <span class="txt-center">Riley's Treat Dispenser</span>
```

Comment:
I think you could have added padding here as well, instead of centering it. Alignments are a bit off when the titles are of various lengths. Like, it is not aligning with the amount.
<img width="502" alt="image" src="https://user-images.githubusercontent.com/38975782/153737201-fb54655d-c729-4b66-a74e-da783499a7ac.png">

Reply:
Thanks for the suggestion, makes the card look much better now !